test/specs/sObjectsDefinitions.e2e.ts

```diff
@@ -0,0 +1,216 @@
+import * as fs from 'fs'; 
+import path from 'path';
+
+describe('SObjects Definitions', async () => {
+  let scratchOrg: ScratchOrg;
+
+  step('Set up the testing environment', async () => {
+    scratchOrg = new ScratchOrg('sObjectsDefinitions', false);
+    await scratchOrg.setUp();
+    const projectPath = scratchOrg.projectFolderPath;
+    const tempFolderPath = scratchOrg.tempFolderPath;
+    const source = path.join(tempFolderPath!, '..', 'test', 'testData', 'CustomSObjects');
+    const destination = path.join(projectPath!, 'force-app', 'main', 'default', 'objects');
+
+    fs.cp(source, destination, { recursive: true }, async (error) => {
+      if (error) {
+        await scratchOrg.tearDown();
+        return error
+      }
+    });
+  });
+
+  step('Check Custom Objects Customer and Product are within objects folder', async () => {
+    const workbench = await browser.getWorkbench();
+    const sidebar = workbench.getSideBar();
+    const content = sidebar.getContent();
+
+    const treeViewSection = await content.getSection(scratchOrg.tempProjectName.toUpperCase());
+    expect(treeViewSection).not.toEqual(undefined);
+
+    const objectTreeItem = await treeViewSection.findItem('objects') as DefaultTreeItem;
+    expect(objectTreeItem).not.toEqual(undefined);
+    await objectTreeItem.select();
+  
+    const customerObjectFolder = await objectTreeItem.findChildItem('Customer__c');
+    expect(customerObjectFolder).not.toEqual(undefined);
+    await customerObjectFolder?.expand();
+    expect(await customerObjectFolder?.isExpanded()).toBe(true);
+    const customerCustomObject = await customerObjectFolder?.findChildItem('Customer__c.object-meta.xml');
+    expect(customerCustomObject).not.toEqual(undefined);
+
+    const productObjectFolder = await objectTreeItem.findChildItem('Product__c');
+    expect(productObjectFolder).not.toEqual(undefined);
+    await productObjectFolder?.expand();
+    expect(await productObjectFolder?.isExpanded()).toBe(true);
+    const productCustomObject = await productObjectFolder?.findChildItem('Product__c.object-meta.xml');
+    expect(productCustomObject).not.toEqual(undefined);
+  });
+    
+  step('Push Source to Org', async () => {
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Scratch Org', 5);
+    await utilities.pause(1);
+
+    const successNotificationWasFound = await utilities.attemptToFindNotification(workbench, 'SFDX: Push Source to Default Scratch Org successfully ran', 10);
+    expect(successNotificationWasFound).toBe(true);
+
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'Starting SFDX: Push Source to Default Scratch Org', 5);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('Pushed Source');
+  });
+
+  step('Refresh SObject Definitions for Custom SObjects', async () => {
```

Comment:
The "Refresh SObject Definitions for Custom SObjects" test is failing for me on my local machine.

```
[chrome 102.0.5005.196 mac os x #0-0] Error: expect(received).not.toEqual(expected) // deep equality
[chrome 102.0.5005.196 mac os x #0-0]
[chrome 102.0.5005.196 mac os x #0-0] Expected: not undefined
[chrome 102.0.5005.196 mac os x #0-0]
[chrome 102.0.5005.196 mac os x #0-0]     at Context.<anonymous> (/Users/jbeeghly/src/github.com/forcedotcom/salesforcedx-vscode-automation-tests-3/test/specs/sObjectsDefinitions.e2e.ts:118:34)
```

Reply:
I ran it and it's all passing on my local.

Reply:
When I run, within /.sfdx/tools there is [sobjects, soqlMetadata, testresults, apex.db]

But when `await toolsTreeItem.getChildren();` is called, only [testresults, apex.db] are returned.